store/etcd/etcd.go

```diff
@@ -470,6 +567,32 @@ func (s *Etcd) List(directory string) ([]*store.KVPair, error) {
 	return kv, nil
 }
 
+// ListExt lists child nodes of a given directory, etcd native extension.
+func (s *Etcd) ListExt(directory string) (*store.KVPairExt, error) {
```

Comment:
Last argument.
Etcd doesn't have lists, it has directories which identified by property `Dir` set to true.
List of things with `Dir` true is like a list of lists which is confusing.